GUI/Main/Main.cs

```diff
@@ -959,8 +959,11 @@ private void launchKSPToolStripMenuItem_Click(object sender, EventArgs e)
             // https://issuetracker.unity3d.com/issues/linux-segmentation-fault-when-running-a-built-project-with-single-instance-argument
```

Comment:
Comment above this could be updated to say "1.8 and 1.9"

Reply:
Went with "to", easier to update for KSP 1.10.